src/dotnet/APIView/APIViewWeb/Languages/LanguageProcessor.cs

```diff
@@ -42,34 +44,67 @@ public override async Task<CodeFile> GetCodeFileAsync(string originalName, Strea
             {
                 await stream.CopyToAsync(file);
             }
+            return await RunParserProcess(originalName, tempDirectory, jsonFilePath, Name);
+        }
 
-            try
+        public string RunProcess(string workingDirectory, string processName, string arguments)
+        {
+            var processStartInfo = new ProcessStartInfo(ProcessName, arguments);
+            string processErrors = String.Empty;
+
+            processStartInfo.WorkingDirectory = workingDirectory;
+            processStartInfo.RedirectStandardError = true;
+            processStartInfo.RedirectStandardOutput = true;
+            processStartInfo.UseShellExecute = false;
```

Comment:
nit: Your pr body says

> Use UseShellExecute and read output streams before calling WaitForExit to avoid deadlocks.

But you're explicitly _disabling_ UseShellExecute. I believe this is the default as well. Just asking for clarity.